code/modules/minimap/minimap.dm

```diff
@@ -64,12 +74,19 @@
 		var/area/A = T.loc
 		return A.station_map_colour
 
-	///Creates a minimap marker from a specified target, icon, and icon state.
-	proc/create_minimap_marker(var/atom/target, var/icon, var/icon_state)
+	///Create an alpha mask to hide anything outside the bounds of the physical map.
+	proc/create_alpha_mask()
+		var/icon/mask_icon = icon('icons/obj/minimap/minimap.dmi', "blank")
+		mask_icon.Scale(300 * src.map_scale, 300 * src.map_scale)
```

Comment:
`300` is a magic number, probably worthwhile to use a (local?) define

Reply:
there's already `world.maxx/y`